### PR TITLE
PLAT-559: sender test

### PR DESCRIPTION
**Pull request by MariaVasilenko**
* updated cases:
https://insolar.testrail.io/index.php?/cases/view/5188
https://insolar.testrail.io/index.php?/cases/view/5196
* added test

### CODE REVIEW

File: ledger-core/virtual/integration/sender_test.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	},
+	{
+		name: "VDelegatedCallRequest",
+		msg:  &payload.VDelegatedCallRequest{},
+	},
+
+	{
+		name: "VFindCallResponse",
+		msg:  &payload.VFindCallResponse{},
+	},
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/sender_test.go:64]: Can you make the same order as in TestRail, please ^^

File: ledger-core/virtual/integration/sender_test.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+}
+
+func TestSender_EqualExpectedVE(t *testing.T) {
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/sender_test.go:67]: TestVirtual_DelegationTokenSender (just proposal)

File: ledger-core/virtual/integration/sender_test.go (hunk 1 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/sender_test.go:62]: done

File: ledger-core/virtual/integration/sender_test.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+	for _, cases := range testCases {
+		t.Run(cases.name, func(t *testing.T) {
+			t.Log(cases.caseId)
+			for _, testMsg := range messagesWithoutToken {
+				t.Run(testMsg.name, func(t *testing.T) {
+
+					mc := minimock.NewController(t)
+
+					server, ctx := utils.NewUninitializedServerWithErrorFilter(nil, t, func(s string) bool {
```
Comment (bigbes) [inline ledger-core/virtual/integration/sender_test.go:83]: But why you don't set error handler here?

Comment (sexyhamster) [reply ledger-core/virtual/integration/sender_test.go:83]: there should be check that only expected errors are skipped

File: ledger-core/virtual/integration/sender_test.go [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+	}{
+		{"Sender is equal expectedVE", "C5188", true},
+		{"Sender is not equal expectedVE", "C5196", false},
+	}
+
+	for _, cases := range testCases {
+		t.Run(cases.name, func(t *testing.T) {
+			t.Log(cases.caseId)
+			for _, testMsg := range messagesWithoutToken {
+				t.Run(testMsg.name, func(t *testing.T) {
```
Comment (ElSacramento) [inline ledger-core/virtual/integration/sender_test.go:79]: Add `defer commontestutils.LeakTester(t)` please

Comment (MariaVasilenko) [reply ledger-core/virtual/integration/sender_test.go:79]: done

File: ledger-core/virtual/integration/sender_test.go [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+						})
+					}
+
+					reflect.ValueOf(testMsg.msg).MethodByName("Reset").Call([]reflect.Value{})
+
+					server.SendPayload(ctx, testMsg.msg.(payload.Marshaler)) // default caller == server.GlobalCaller()
+
+					server.WaitIdleConveyor()
+
+					if cases.senderIsEqualExpectedVE || testMsg.name == "VFindCallResponse" {
```
Comment (bigbes) [inline ledger-core/virtual/integration/sender_test.go:129]: please don't operate on testMsg.name in tests

File: ledger-core/virtual/integration/sender_test.go (hunk 2 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/sender_test.go]: done

File: ledger-core/virtual/integration/sender_test.go (hunk 5 above)
Comment (MariaVasilenko) [reply ledger-core/virtual/integration/sender_test.go:129]: Why?

**Pull request merged**